AZURE-original/index.html

```diff
@@ -189,7 +189,7 @@ <h2>On-premises,hybrid, multicloud, or at the edge—
             <div class="grid">
                 <div class="card-1">
```

Comment:
try to use meaningful names for class names, in case you are using numbers try to use words instead  of numericals